### DISCUSSION TITLE
Accessing temperature and humidity sensor through a supported zigbee gateway

### DISCUSSION THREAD

**Discussion by awakenrz** [merdok/homebridge-miot#376] [Q&A]
I read that in 1.4.0, support of subdevices through a zigbee gateway is added.
Does that mean I could access Xiaomi temperature and humidity sensors?

**Comment by merdok** [comment]
Yes, you could do that.
Someone described it very detailed here: https://github.com/merdok/homebridge-miot/discussions/348
Basically you need to specify the token and ip of your gateway, and you need to specify the deviceId and model of your subdevice. That should allow you to control sub devices.
